### DISCUSSION TITLE
Thermostate werden nicht mehr Angezeigt.

### DISCUSSION THREAD

**Discussion by jeng37** [Blackymas/NSPanel_HA_Blueprint#1767] [General]
Hallo,
Ich habe bemerkt dass auf den Button pages keine Thermostate angezeigt werden. Übrigens nirgens wo werden diese angezeigt. Ist das normal?
System ist alles auf dem letztetn stand! Läuft auf einem rpi 4.
Thermostate sind über zigbee2mqtt eingebunden. Es lief ja alles bis dahin wo das letzte update kamm.
Wenn ich auf den Button Pages einen Thermosat aus wähle, wird er nicht angezeigt, die Seite bleibt einfach nur leer.
Habe ich irgendwas verpasst?

**Comment by edwardtfn** [comment]
It should work.
Can you find any error on Home Assistant logs (Settings > System > Logs)?

**Reply by jeng37** [reply]
Nope, no errors at all. evrything ist empty. hmm?

**Comment by jeng37** [comment]
![Screenshot 2024-02-05 122227](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/12857791/5a54875e-1242-47b7-b441-08a428b04386)

as you see...

**Comment by jeng37** [comment]
![Screenshot 2024-02-05 122520](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/12857791/1a8dda8b-1930-44c5-890f-010837a820f2)
![Screenshot 2024-02-05 122539](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/12857791/785e9beb-e0fc-42a3-b9e5-4d16e215a079)

all ok, and i can select the items on Button page, but nothing appears on nspanel.....

**Comment by jeng37** [comment]
https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/12857791/65e9f846-8d73-4aba-8962-9b91178eb912

little video

**Comment by jeng37** [comment]
![Screenshot 2024-02-05 122824](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/12857791/c101f8a7-bcf0-4bbf-ac62-12d3b6260487)
![Screenshot 2024-02-05 122937](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/12857791/a0cb7601-bc7e-4875-96b4-f1510255373f)

**Comment by jeng37** [comment]
I can't find any errors.
I have already deleted the entire config, re-imported the blueprint and reconfigured it. But nothing... always the same scheme.

**Comment by edwardtfn** [comment]
This is odd. Looks like the Blueprint is crashing when constructing that page, bit a crash should be registered at your logs.

**Reply by edwardtfn** [reply]
Which version are you running?

**Comment by jeng37** [comment]
Bluetpint v4.2.2.

**Reply by edwardtfn** [reply]
You should update to v4.2.4 (latest stable) there were fixes around this:

- https://github.com/Blackymas/NSPanel_HA_Blueprint/releases/tag/v4.2.3

**Comment by jeng37** [comment]
![Screenshot 2024-02-05 125442](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/12857791/a0d8398d-6d3c-42f7-974e-95c176ed01d0)
![Screenshot 2024-02-05 125601](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/12857791/aee31b53-14dc-41b1-8e45-683bf8e270a9)

**Comment by jeng37** [comment]
nspanel log.
[logs_nspanel-living_logs.txt](https://github.com/Blackymas/NSPanel_HA_Blueprint/files/14165271/logs_nspanel-living_logs.txt)

**Comment by jeng37** [comment]
Should I open an issue?

**Reply by edwardtfn** [reply]
Please update to v4.2.4 first and if this problem persist please report it as a bug.

**Comment by jeng37** [comment]
Hey Man, thx,
with v4.2.4 it works again!
Thank you!

**Reply by edwardtfn** [reply]
Nice!!
